app/client/components/List.js

```diff
@@ -4,21 +4,25 @@ import React from 'react';
 import ListGroup from 'react-bootstrap/lib/ListGroup';
 import ListItem from 'components/ListItem';
 
-export default class Items extends React.Component {
+export default class List extends React.Component {
 
     static propTypes = {
         items: React.PropTypes.array.isRequired,
+        mode: React.PropTypes.string,
```

Comment:
Can we implement a more string validation here?
`React.PropTypes.oneOf([...])``

We will support only two or three very specific modes so we can check that the given value is part of an enumeration